upup/models/cloudup/resources/addons/networking.projectcalico.org/k8s-1.6.yaml.template

```diff
@@ -5,46 +5,35 @@ metadata:
   name: calico-config
   namespace: kube-system
 data:
-  # The calico-etcd PetSet service IP:port
+  # etcd servers
   etcd_endpoints: "{{ $cluster := index .EtcdClusters 0 -}}
                       {{- range $j, $member := $cluster.Members -}}
                           {{- if $j }},{{ end -}}
                           http://etcd-{{ $member.Name }}.internal.{{ ClusterName }}:4001
                       {{- end }}"
 
-  # Configure the Calico backend to use.
+# Configure the Calico backend to use.
```

Comment:
commend indentation is off :)

Reply:
😆 yes it is ... fixing.  After the push `/lgtm`?